wakuv2/nwaku.go

```diff
@@ -0,0 +1,58 @@
+package wakuv2
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"strconv"
+)
+
+type NwakuInfo struct {
+	ListenAddresses []string `json:"listenAddresses"`
+	EnrUri          string   `json:"enrUri"`
+}
+
+func GetNwakuInfo(host *string, port *int) (NwakuInfo, error) {
```

Comment:
Maybe we should keep `GetNwakuInfo` in `wakuv2/waku_test.go`?
Since it's not really a public function, should only be used in tests 🤔

Reply:
I was trying to use this in protocol/messenger_storenode_request_test.go but noticed that if i kept it in waku_test.go I couldn't import it from other packages hence why I moved it to a separate non test file